react/ProductSummaryCustom.tsx

```diff
@@ -162,44 +168,52 @@ const ProductSummaryCustom = ({
           </section>
         </ProductPriceSimulationWrapper>
       </ProductContextProvider>
-    </ProductSummaryContext.Provider>
+    </LocalProductSummaryContext.Provider>
   )
 }
 
-ProductSummaryCustom.propTypes = {
-  /** Product that owns the informations */
-  product: productShape,
+interface Props {
+  product: ProductSummaryTypes.Product
   /** Function that is executed when a product is clicked */
-  actionOnClick: PropTypes.func,
-  children: PropTypes.node,
-  containerRef: PropTypes.oneOfType([
```

Comment:
What did this prop do and why could it be removed?

Reply:
It was not being used 🤷‍♂️

Reply:
It was sent to the context via the `...rest`, wasn't it?

Reply:
I've found a changelog entry saying it was removed on version 2.51.2. Now I'm convinced it's safe 😄